docs/docs/whatsnew.md

```diff
@@ -1,5 +1,30 @@
 # <b>Important updates</b>
 
+## v1.2.25
+
+### Configuration of `maven` builder has changed. 
+As of version 1.2.25, the `mvn -B clean package` command is used where the `maven` builder is configured for building a module or in a global build step. 
+
+The `-B` parameter is added to `maven` builder's build command which will start maven in a interactive mode.
```

Comment:
The `-B` parameter is added to the `maven` builder's build command, which will start maven in interactive mode.

(Does the build command start maven in interactive mode? or the -B parameter? Just trying to understand how to say this most clearly.)

Reply:
By adding -B parameter, the "maven clean package" command will start in interactive mode

Reply:
Ok. It's better this way. 
By adding the -B parameter, the "maven clean package" command will start in interactive mode.

Reply:
ok, Done